rclpy/services/minimal_client/client.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+
+import rclpy
+
+from example_interfaces.srv import AddTwoInts
+
+
+def main(args=None):
+    rclpy.init(args)
+    node = rclpy.create_node('minimal_client')
+    cli = node.create_client(AddTwoInts, 'add_two_ints')
+
+    req = AddTwoInts.Request()
+    req.a = 41
+    req.b = 1
+    # wait for connection to be established
+    # (no wait for service in Python yet)
```

Comment:
I would recommend a TODO here then, so we remember to replace this with wait for service later. Or if there is a ticket to add wait for service in Python, you could link to this line of this file (at a specific commit) as a place to update after it is added. You might want to do both.